joplin/api/schema.py

```diff
@@ -104,7 +104,7 @@ class ServicePageNode(DjangoObjectType):
     related = graphene.List('api.schema.ServicePageNode')
 
     def resolve_related(self, resolve_info, *args, **kwargs):
-        return self.topic.services.exclude(id=self.id)
+        return self.topic.servicepage_set.exclude(id=self.id)
```

Comment:
now that I've seen the Janis PR, why do we need this change?

Reply:
Hmm--- I'll try to explain this but -- here's a [stackoverflow](https://stackoverflow.com/questions/2642613/what-is-related-name-used-for-in-django) that probably does a better job.

Django allows for reverse lookups of properties on models that are foreign keys to other models. So for example, All JanisPages have a lookup to a topic model, so we can look up all JanisPages that have a particular topic. But since we inherit from JanisPages to create child pagetypes (currently includes ServicePage and ProcessPage) Django needs a way to distinguish between the two types. So give me all ServicePages for a particular topic or give me all ProcessPages for a particular topic. To do this, Django automatically creates a property on a topic model to for this look up by using the <model name>_set. So to access all service pages with a particular topic we would do topic.servicepage_set and similarly for process pages ... topic.processpage_set. So both of those properties are created on the fly by Django. 

The reason for the change in Janis is related to the definition of the topic property in the base JanisPage model. Since we previously didn't inherit the topic property from a base JanisPage model, we could specify the name we wanted for this property (rather than letting Django create it for us with <model name>_set). We did this by specifying a 'related_name' when defining the topic property on each page model as follows ...
```
class ServicePage(Page):
    topic = models.ForeignKey(	
        'base.Topic',	
        on_delete=models.PROTECT,	
        related_name='services',	
        verbose_name='Select a Topic',	
    )
...
```
But since we're now inheriting topic in child page types, I allow Django to autogenerate the topic lookup property name so it will work for all children pages types that inherit from JanisPage.